test/check-application

```diff
@@ -306,20 +325,71 @@ class TestNavigator(testlib.MachineCase):
         b.click("button.pf-m-primary")        
         b.wait_visible("[data-item='newdir']")
 
-        # Delete folder from context menu
+        # Rename folder from context menu
         b.mouse("[data-item='newdir']", "contextmenu")
-        b.wait_in_text(".context-menu li:nth-child(2) button", "Delete directory")
+        b.wait_in_text(".context-menu li:nth-child(2) button", "Rename directory")
+        b.click(".context-menu button:contains('Rename directory')")
+        b.set_input_text("#rename-item-input", "newdir1")
+        b.click("button.pf-m-primary")
```

Comment:
Can you please in followup add some more readable and unique selector here? and any other possible place that you use this selector. An 'id' in the element in the JSX code and reuse of the id here would be ideal.